src/components/HeaderInside/index.tsx

```diff
@@ -69,19 +71,35 @@ const HeaderInside: React.FC = () => {
           </LinkContainer>
         )}
       </Content>
+      <section className="box-localizacao">
+        <img src={MapIcon} alt="Icone-localizacao" />
+        <p>Enviar para Jonisson Gomes da Silva </p>
+        <strong>, Recife - 53610</strong>
+      </section>
       <Menu>
         <MenuContent>
-          <div />
-          <ul>
-            <li>Black Friday</li>
-            <li>Bombando</li>
-            <li>Categorias</li>
-            <li>Sobre</li>
-          </ul>
+          <section className="box-localizacao">
+            <img src={MapIcon} alt="Icone-localizacao" />
```

Comment:
Tem duas sections completamente iguais. Transforma elas em um component apenas.

Reply:
São iguais, porém 1 para um determinado tamanho de tela e outra para outro tipo de tela, então to tratando a visibilidade apenas ai.
Como são utilizados apenas ai então acho que não vale apena transformar em componente, caso haja uso em outra página eu transformo.

Reply:
Mas o componente não deixa de ser o mesmo. Tu só ta escondendo um ou outro com base no tamanho da tela. Nesse sentido, ainda está sendo repetição de código, então acredito que valha a pena transformar em um componente.